mmpose/datasets/pipelines/shared_transform.py

```diff
@@ -406,6 +406,50 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class MultiItemProcess:
+    """Process each item and merge multi-item results to lists.
+
+    Args:
+        pipeline (dict): Dictionary to construct pipeline for a single item.
+    """
+
+    def __init__(self, pipeline):
+        self.pipeline = Compose(pipeline)
+
+    def __call__(self, results):
+        results_ = {}
+        for idx, result in results.items():
+            single_result = self.pipeline(result)
+            for k, v in single_result.items():
+                if k in results_:
+                    results_[k].append(v)
+                else:
+                    results_[k] = [v]
+
+        return results_
+
+
+@PIPELINES.register_module()
+class MultiItemDeduplicate:
```

Comment:
The class name could be a little confusing. What about `DiscardDuplicatedItems`?

Reply:
Nice suggestion.